rustls/src/msgs/handshake.rs

```diff
@@ -388,22 +388,18 @@ impl AsRef<[u8]> for SingleProtocolName {
 
 // --- TLS 1.3 Key shares ---
 #[derive(Clone, Debug)]
-pub struct KeyShareEntry {
+pub(crate) struct KeyShareEntry {
```

Comment:
I think these privatizations could/should be part of the earlier commit that makes their being public unnecessary?